tutorials/understanding-gov/index.md

```diff
@@ -0,0 +1,324 @@
+
+* **Quorum:** Quorum is defined as the minimum percentage of voting power that needs to be cast on a proposal for the result to be valid. If the quorum is not reached, the proposal is rejected.
+
+* **Voting options:** Voters can choose between `Yes`, `No`, `NoWithVeto` and `Abstain`. `NoWithVeto` allows the voter to cast a `No` vote, but also to veto the proposal. If a proposal is vetoed, it is automatically rejected and the deposit burned. `Abstain` allows the voter to abstain from voting. With a majority of `Yes` the proposal pass and its messages are executed.
+
+More information about the governance concepts can be found in the [Cosmos SDK documentation](https://docs.cosmos.network/v0.46/modules/gov/01_concepts.html).
+
+## Requirements
+
+The gov module has been introduced in the [v0.46.0 release](https://docs.cosmos.network/v0.46/modules/gov/) of the Cosmos SDK. To follow the tutorial, you must use the binary of a chain with the gov module, using a v0.46+ version of the SDK. For demonstration purposes, you will use `simd`, the simulation app of the Cosmos SDK.
+
+To install `simd`, first clone the Cosmos SDK GitHub repository and checkout the right version:
+
+```sh
+$ git clone https://github.com/cosmos/cosmos-sdk --depth=1 --branch v0.46.3
+```
+
+We are installing `v0.46.3` because this version added the command `draft-proposal`. We will explain later what it does.
+
+Go to the cloned directory:
+
+```sh
+$ cd cosmos-sdk
+```
+
+Install `simd`:
+
+```sh
+$ make install
+```
+
+Make sure the installation was successful:
+
+```sh
+$ simd version
+```
+
+The version number should be greater than or equal to `0.46.3`.
+
+## Configuration
+
+<HighlightBox type="tip">
+
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. You can skip to the next section or remove the chain directory (`rm -rf ~/.simapp`).
+
+</HighlightBox>
+
+For configuring `simd`, you have to set the chain ID and the keyring backend.
+
+```sh
+$ simd config chain-id demo
+$ simd config keyring-backend test
+```
+
+Secondly, you need to add keys for chain users. Call them Alice and Bob:
+
+```sh
+$ simd keys add alice
+$ simd keys add bob
+```
+
+With `simd keys list` you can verify that your two users have been added.
+
+<HighlightBox type="tip">
+
+To avoid having to copy and paste the user addresses, now is a good time to export the user keys to variables that you can access and use for this tutorial.
+
+</HighlightBox>
+
+```sh
+$ export ALICE=$(simd keys show alice --address)
+$ export BOB=$(simd keys show bob --address)
+```
+
+Now you are ready to fund Alice and Bob's respective accounts and use the Alice account as validator:
+
+```sh
+$ simd init test --chain-id demo
+```
+
+The default voting period is **172800s**. It is too long to wait for the tutorial, so you will change it to **180s**. To do so, you need to edit the `~/.simapp/config/genesis.json` file:
```

Comment:
If you think it is useful, I got this one-liner that can update in place https://github.com/cosmos/sdk-tutorials/blob/master/academy/3-my-own-chain/migration.md?plain=1#L1604

Reply:
I was sure I saw that somewhere, but couldn't find it back! Thanks :)